Apps/phpanorama/panorama.json

```diff
@@ -60,7 +60,7 @@
         {
             "action": "block url",
             "description": "Block an URL",
-            "verbose": "This action does the following to block a URL:<ul><li>Create an URL Filtering profile object named '<b>Phantom URL List for [device_group]</b>' containing the URL to block.</br>If the profile is already present, then it will be updated to include the URL to block.</li><li>Re-Configure the policy (specified in the <b>policy_name</b> parameter) to use the created URL Filtering profile.</br>If the policy is not found on the device, the action will return an error.</li><li>The action then proceeds to <b>commit</b> the changes to Panorama, followed by a commit to the device group. If the device group happens to be <b>shared</b>, then a commit will be sent to all the device groups belonging to it.</li></ul>",
+            "verbose": "This action does the following to block a URL:<ul><li>Create an URL Filtering profile object named '<b>Phantom URL List for [device_group]</b>' containing the URL to block.</br>If the profile is already present, then it will be updated to include the URL to block. IMPORTANT: For Version 9 and above, a URL Filtering profile no longer includes allow-list/block-list. The official workaround is to use a Custom URL category instead. Therefore, we create a new Custom URL category with the same name as the profile and link it to the profile. Then, We configure the profile to block the URL category on both 'SITE ACCESS' and 'USER CREDENTIAL SUBMISSION' columns.</li><li>Re-Configure the policy (specified in the <b>policy_name</b> parameter) to use the created URL Filtering profile.</br>If the policy is not found on the device, the action will return an error.</li><li>The action then proceeds to <b>commit</b> the changes to Panorama, followed by a commit to the device group. If the device group happens to be <b>shared</b>, then a commit will be sent to all the device groups belonging to it.</li></ul>",
```

Comment:
For more info: https://10.1.66.58:9999/docs/app_reference/panorama_3d88e427-0e61-4311-a613-bbddeac532e0#block-url

![image](https://user-images.githubusercontent.com/82124762/129091468-7c087064-9434-40ab-9b87-ef4b195b4e80.png)